Packs/CommonScripts/Scripts/ParseEmailFilesV2/ParseEmailFilesV2.py

```diff
@@ -0,0 +1,147 @@
+    if email_file_name:
+        md = f"### {email_file_name}\n"
+
+    if print_only_headers:
+        return tableToMarkdown(f"Email Headers: {email_file_name}", email_data.get('HeadersMap'))
+
+    if parent_email_file:
+        md += f"### Containing email: {parent_email_file}\n"
+
+    md += u"* {0}:\t{1}\n".format('From', email_data.get('From') or "")
+    md += u"* {0}:\t{1}\n".format('To', email_data.get('To') or "")
+    md += u"* {0}:\t{1}\n".format('CC', email_data.get('CC') or "")
+    md += u"* {0}:\t{1}\n".format('Subject', email_data.get('Subject') or "")
+    if email_data.get('Text'):
+        text = email_data['Text'].replace('<', '[').replace('>', ']')
+        md += u"* {0}:\t{1}\n".format('Body/Text', text or "")
+    if email_data.get('HTML'):
+        md += u"* {0}:\t{1}\n".format('Body/HTML', email_data['HTML'] or "")
+
+    md += u"* {0}:\t{1}\n".format('Attachments', email_data.get('Attachments') or "")
+    md += u"\n\n" + tableToMarkdown('HeadersMap', email_data.get('HeadersMap'))
+    return md
+
+
+def save_file(file_name, file_content) -> str:
+    """
+    save attachment to the war room and return the file internal path.
+
+    Args:
+      file_name (str): The name of the file to be created.
+      file_content (str/bytes): the file data.
+
+    Returns:
+      str: the file internal path
+
+    """
+    created_file = fileResult(file_name, file_content)
+    file_id = created_file.get('FileID')
+    attachment_internal_path = demisto.investigation().get('id') + '_' + file_id
+    demisto.results(created_file)
+
+    return attachment_internal_path
+
+
+def extract_file_info(entry_id: str) -> tuple:
+    """
+    extract from the entry id the file_type, file_path and file_name.
+
+    Args:
+      entry_id (str): The entry id.
+
+    Returns:
+        file_type(str): the file mime type.
+        file_path(str): the file path.
+        file_name(str):the file name.
+    """
+    file_type = ''
+    file_path = ''
+    file_name = ''
+    try:
+        result = demisto.executeCommand('getFilePath', {'id': entry_id})
+        if is_error(result):
+            return_error(get_error(result))
+
+        file_path = result[0]['Contents']['path']
+        file_name = result[0]['Contents']['name']
+        result = demisto.executeCommand('getEntry', {'id': entry_id})
+        if is_error(result):
+            return_error(get_error(result))
+
+        file_metadata = result[0]['FileMetadata']
+        file_type = file_metadata.get('info', '') or file_metadata.get('type', '')
+
+    except Exception as ex:
+        return_error(
+            "Failed to load file entry with entry id: {}. Error: {}".format(
+                entry_id, str(ex) + "\n\nTrace:\n" + traceback.format_exc()))
+
+    return file_type, file_path, file_name
+
+
+def main():
+    args = demisto.args()
+    entry_id = args.get('entryid')
+    max_depth = arg_to_number(args.get('max_depth', '3'))
+    if max_depth is None or max_depth < 1:
```

Comment:
Isn't it better to check:
```suggestion
    if not max_depth or max_depth < 1:
```